Reason: Adjust policies directory for a system-wide installation
Upstream: No

diff --git a/ssh-audit.py b/ssh-audit.py
index 3884996..52f7d81 100755
--- a/ssh-audit.py
+++ b/ssh-audit.py
@@ -3399,9 +3399,7 @@ def evaluate_policy(aconf: AuditConf, banner: Optional['SSH.Banner'], client_hos
 
 def list_policies() -> None:
 
-    # Get a list of all the files in the policies sub-directory, relative to the path of this script.
-    installed_dir = os.path.dirname(os.path.abspath(__file__))
-    policies_dir = os.path.join(installed_dir, 'policies')
+    policies_dir = '/usr/share/ssh-audit/policies'
 
     # If the path is not a directory, print a useful error and exit.
     if not os.path.isdir(policies_dir):
